src/Components/drills/FrisbeeDrillIllustration.js

```diff
@@ -88,14 +89,26 @@ const FrisbeeDrillIllustration = (props) => {
     );
   };
 
-  const renderPagination = () => {
+  const renderCounter = (index) => {
+    if (props.drill.steps.length !== 1) {
+      return (
+        <View style={styles.counter}>
+          <Text style={styles.textCounter}>
+            {index + 1}/{props.drill.steps.length}
+          </Text>
+        </View>
+      );
+    }
+  };
+
+  const renderPagination = (index) => {
     return (
       <Pagination
         dotsLength={props.drill.steps.length}
-        activeDotIndex={activeIndex}
+        activeDotIndex={index}
         containerStyle={{
-          backgroundColor: theme.BACKGROUND_COLOR_LIGHT,
```

Comment:
I think your bug came from this line

It should be better with a transparent background